python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -331,33 +376,38 @@ def update_psf_stats(
         # 750bffe6620e565bda731add1509507f5c40c8bb/src/PsfFlux.cc#L112
         summary.psfArea = float(np.sum(im.array)/np.sum(im.array**2.))
 
-        if image_mask is not None:
-            psfApRadius = max(self.config.minPsfApRadiusPix, 3.0*summary.psfSigma)
-            self.log.debug("Using radius of %.3f (pixels) for psfApFluxDelta metric", psfApRadius)
-            psfTraceRadiusDelta, psfApFluxDelta = compute_psf_image_deltas(
-                image_mask,
-                psf,
-                sampling=self.config.psfGridSampling,
-                ap_radius_pix=psfApRadius,
-                bad_mask_bits=self.config.psfBadMaskPlanes
-            )
-            summary.psfTraceRadiusDelta = float(psfTraceRadiusDelta)
-            summary.psfApFluxDelta = float(psfApFluxDelta)
-            if image_ap_corr_map is not None:
-                if self.config.psfApCorrFieldName not in image_ap_corr_map.keys():
-                    self.log.warn(f"{self.config.psfApCorrFieldName} not found in "
-                                  "image_ap_corr_map.  Setting psfApCorrSigmaScaledDelta to NaN.")
-                    psfApCorrSigmaScaledDelta = nan
-                else:
-                    image_ap_corr_field = image_ap_corr_map[self.config.psfApCorrFieldName]
-                    psfApCorrSigmaScaledDelta = compute_ap_corr_sigma_scaled_delta(
-                        image_mask,
-                        image_ap_corr_field,
-                        summary.psfSigma,
-                        sampling=self.config.psfGridSampling,
-                        bad_mask_bits=self.config.psfBadMaskPlanes,
-                    )
-                summary.psfApCorrSigmaScaledDelta = float(psfApCorrSigmaScaledDelta)
+        if self.config.psfGridSampling is None:
+            self.log.info("Note: not computing grid-based PSF & ApCorr model fidelity metrics. Setting "
```

Comment:
Not sure what "model fidelity" means here?

Reply:
Why not be explicit about it? "PSF & ApCorr model minimum/maximum range metrics"? From the docstrings on those functions, that seems like what they are, no?